qrand.go

```diff
@@ -66,6 +68,25 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
+type ΨSource struct{}
+
+var qSource ΨSource
+
+func (s *ΨSource) Seed(seed int64) {}
```

Comment:
Let's comment here to explain why the method is empty (there's nothing to seed!)